src/main/java/com/openshift/jenkins/plugins/pipeline/MessageConstants.java

```diff
@@ -0,0 +1,60 @@
+package com.openshift.jenkins.plugins.pipeline;
+
+public class MessageConstants {
+
+public static final String CANNOT_GET_CLIENT = "\n\nExiting \"%s\" unsuccessfully; a client connection to \"%s\" could not be obtained.";
+public static final String BAD_BUILD_STATE = "\n\nExiting \"%s\" unsuccessfully; build \"%s\" has completed with status:  [%s].";
+public static final String GOOD_BUILD_WITH_DEPLOY = "\n\nExiting \"%s\" successfully; build \"%s\" has completed with status:  [Complete].  All deployments with ImageChange triggers based on this build's output triggered off of the new image.";
+public static final String GOOD_BUILD_BAD_DEPLOY = "\n\nExiting \"%s\" unsuccessfully; build \"%s\" has completed with status:  [Complete]. However, not all deployments with ImageChange triggers based on this build's output triggered off of the new image.";
+public static final String GOOD_BUILD_IGNORE_DEPLOY = "\n\nExiting \"%s\" successfully; build \"%s\" has completed with status:  [Complete].";
+public static final String START_BUILD_CANCEL = "\n\nStarting the \"%s\" action for build config \"%s\" from the project \"%s\".";
+public static final String CANCEL_SINGLE_BUILD = "  Cancelled build \"%s\".";
+public static final String EXIT_BUILD_CANCEL = "\n\nExiting \"%s\" successfully with %d builds cancelled.";
+public static final String START_BUILD = "\n\nStarting the \"%s\" step with build config \"%s\" from the project \"%s\".";
+public static final String BUILD_CANNOT_START = "\n\nExiting \"%s\" unsuccessfully; a build could not be started";
```

Comment:
I guess i'm not clear on under what conditions this message is thrown.  I was thinking you got an error back from the api call.  If this is a case where the build itself was successfully created and then entered some sort of error state (due to conditions in the master/cluster), then this is fine, the user can go inspect the build itself for more details.